### PR TITLE
Update the linkbot

### PR DESCRIPTION
**Pull request by oleorhagen**
Both with new commit setup:
* Sign and author with `mender-test-bot <mender-test-bot@northern.tech>`
* Update the RPi parser, due to the naming semantics of the raspbian index
* Also, run the bot weekly at night sunday -> monday, instead of day monday

### CODE REVIEW

**Comment by oleorhagen** [comment]
@lluiscampos you can ignore the `javascript` commit

File: .github/workflows/automatically-update-test-links.yml [hunk 1]
```diff
@@ -25,7 +25,14 @@ jobs:
             title: "Image-Bot: New images available"
             body: |
                 New images are available for update
-            commit-message: Image-Bot - Image updates
+            commit-message: |
+                Image-Bot - Image updates
+
+                Changelog: Title
+            committer: mender-test-bot <mender-test-bot@mender.io>
+            author: mender-test-bot <mender-test-bot@mender.io>
```
Comment (lluiscampos) [inline .github/workflows/automatically-update-test-links.yml:33]: This e-mail address does not exist, maybe use
```suggestion
            committer: mender-test-bot <mender@northern.tech>
            author: mender-test-bot <mender@northern.tech>
```

**Review by lluiscampos — CHANGES_REQUESTED** [review]
Looks good to me otherwise.

File: .github/workflows/automatically-update-test-links.yml (hunk 1 above)
Comment (oleorhagen) [reply .github/workflows/automatically-update-test-links.yml:33]: Are you really sure this should have a `Changelog`? :thinking: 
It just seems a bit out of place to me

Comment (lluiscampos) [reply .github/workflows/automatically-update-test-links.yml]: While I don't have an strong opinion, I don't think it deserves a changelog, no. There is absolutely nothing changing product code, only testing images. I didn't comment as I thought you had a reason for it and I wanted to respect that :)

Comment (oleorhagen) [reply .github/workflows/automatically-update-test-links.yml]: I just thought I remembered it was asked for previously, but just a brain-fart appareantly. 

Thx

Comment (lluiscampos) [reply .github/workflows/automatically-update-test-links.yml]: Mmm. I did ask about a Changelog tag, in order to pass the commit checks, but I meant "Changelog: None" kind of tag. Maybe it was my bad there, then.

Comment (oleorhagen) [reply .github/workflows/automatically-update-test-links.yml]: And now, the :heavy_check_mark: ?

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**